docs/Catalog/RemovingStacks.md

```diff
@@ -0,0 +1,61 @@
+<!-- markdownlint-disable-next-line first-line-h1 -->
+From time to time, you may need to remove a Quilt stack from your AWS account.
+To do this, you will need to delete the CloudFormation template that created the
```

Comment:
template->stack

"you will need to delete the CloudFormation *stack* that created..."

Reply:
```suggestion
To do this, you will need to delete the CloudFormation stack that created the
```